contracts/conwaygol.scrypt

```diff
@@ -2,18 +2,17 @@ import "arrayUtil.scrypt";
 
 // Conway Game Of Life on a board of N * N
 contract GameOfLife {
-
-
-
     static const int N = 5;
     // effctively we play on a grid of (N+2) * (N+2) without handling boundary cells
     static const int BOARD_SIZE = GameOfLife.N + 2;
-    static bytes LIVE = b'01';
-    static bytes DEAD = b'00';
+    //static const int XY = BOARD_SIZE * BOARD_SIZE;  <- error: invalid array size when run #serializeState
```

Comment:
This is bcoz array size must be known at compile time https://scryptdoc.readthedocs.io/en/latest/ctc.html